software/utils/rpi_config/bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Fail if any command fails
+set -e
+set -o pipefail
+
+echo "==============================================================================="
+echo "=================== RespiraWorks Ventilator bootstrapper ======================"
+echo "==============================================================================="
+echo " "
+echo "  -- clones the repository"
+echo "  -- configures Raspberry Pi:"
+echo "     * no screensaver or screen blanking"
+echo "     * no splash on bootup"
+echo "     * serial interface enabled for GUI<->controller comms"
```

Comment:
Could we have this enable ssh access as well or do you think this is a security hazard?
```
sudo systemctl enable ssh
sudo systemctl start ssh
```
should do the trick (i used raspi-config on my end but this isn't scriptable).

Reply:
Won't do this for now. Security, but I am also not using this. This can be a separate PR.